sdk/core/core-http/review/core-http.api.md

```diff
@@ -641,7 +641,7 @@ export interface RequestPrepareOptions {
     baseUrl?: string;
     body?: any;
     bodyIsStream?: boolean;
-    deserializationMapper?: object;
+    deserializationMapper?: Record<string, unknown>;
```

Comment:
Is this necessary?

Reply:
`object` is rejected  by "@typescript-eslint/ban-types"